src/pymortests/base.py

```diff
@@ -92,3 +94,21 @@ def _inner_wrapper(*args, **kwargs):
             return hypothesis.settings(deadline=dl)(func)(*args, **kwargs)
         return _inner_wrapper
     return _outer_wrapper
+
+
+def skip_if_missing(module_name):
+    """Wrapper for requiring certain modules on tests."""
+    def _outer_wrapper(func):
+        @wraps(func)
+        def _inner_wrapper(*args, **kwargs):
+            try:
+                module = importlib.import_module(module_name)
+            except ImportError as ie:
+                if not os.environ.get('DOCKER_PYMOR', False):
+                    skip_string = 'skipped test due to missing module ' + module_name
+                    skip(skip_string)
```

Comment:
When relying on the config module instead of raising the exception again here, this would probably be a straight `pytest.fail` ?

Reply:
I'm not 100% sure if the current implementation does what you require, but I changed it to just catch the exception and fail the test.

Reply:
Right, I did not communicate that well, sorry. 
I was thinking of just using `if config.HAVE_`, but using `config.require` is actually more elegant here. So I'm happy with this :+1: